test/functional/test_runner.py wallet_listtransactions.py fails when run individually

When I run complete functional tests using `test/functional/test_runner.py`, all tests pass. Also, when I run `test/functional/wallet_listtransactions.py` individually, it is successfull.
```
$ test/functional/wallet_listtransactions.py 
2018-11-15T22:08:25.288000Z TestFramework (INFO): Initializing test directory /tmp/test60gysa60
2018-11-15T22:08:51.034000Z TestFramework (INFO): Stopping nodes
2018-11-15T22:08:51.998000Z TestFramework (INFO): Cleaning up /tmp/test60gysa60 on exit
2018-11-15T22:08:51.998000Z TestFramework (INFO): Tests successful
```
But when I run `test/functional/test_runner.py wallet_listtransactions.py`, it always fails with the "AssertionError: Mempool sync timed out".
```
$ LC_ALL=lv_LV.UTF-8 test/functional/test_runner.py wallet_listtransactions.py
Temporary test directory at /tmp/test_runner_₿_🏃_20181116_001822
WARNING! There is already a bitcoind process running on this system. Tests may fail unexpectedly due to resource contention!
1/1 - wallet_listtransactions.py failed, Duration: 84 s                                                                                                                 

stdout:
2018-11-15T22:18:22.640000Z TestFramework (INFO): Initializing test directory /tmp/test_runner_₿_🏃_20181116_001822/wallet_listtransactions_0
2018-11-15T22:19:45.625000Z TestFramework (ERROR): Assertion failed
Traceback (most recent call last):
  File "XXX/bitcoin/test/functional/test_framework/test_framework.py", line 171, in main
    self.run_test()
  File "XXX/bitcoin/test/functional/wallet_listtransactions.py", line 36, in run_test
    self.sync_all()
  File "XXX/bitcoin/test/functional/test_framework/test_framework.py", line 372, in sync_all
    sync_mempools(group)
  File "XXX/bitcoin/test/functional/test_framework/util.py", line 408, in sync_mempools
    raise AssertionError("Mempool sync timed out:{}".format("".join("\n  {!r}".format(m) for m in pool)))
AssertionError: Mempool sync timed out:
  {'88a5f9e47a8617c29eb2330d06cb8af84e30dbcf8276c018b522af9d2cfbf297'}
  set()
2018-11-15T22:19:45.682000Z TestFramework (INFO): Stopping nodes
2018-11-15T22:19:46.646000Z TestFramework (WARNING): Not cleaning up dir /tmp/test_runner_₿_🏃_20181116_001822/wallet_listtransactions_0
2018-11-15T22:19:46.646000Z TestFramework (ERROR): Test failed. Test logging available at /tmp/test_runner_₿_🏃_20181116_001822/wallet_listtransactions_0/test_framework.log
2018-11-15T22:19:46.648000Z TestFramework (ERROR): Hint: Call XXX/bitcoin/test/functional/combine_logs.py '/tmp/test_runner_₿_🏃_20181116_001822/wallet_listtransactions_0' to consolidate all logs


stderr:



TEST                       | STATUS    | DURATION

wallet_listtransactions.py | ✖ Failed  | 84 s

ALL                        | ✖ Failed  | 84 s (accumulated) 
Runtime: 84 s
```

Looked at logs mentioned in output, but could not find anything suspicous there.

Is it a bug or feature? :) Any ideas?

It is with current master.

--- a/test/functional/test_framework/test_framework.py
+++ b/test/functional/test_framework/test_framework.py
@@ -371,21 +371,6 @@ class BitcoinTestFramework(metaclass=BitcoinTestMetaClass):
             sync_blocks(group)
             sync_mempools(group)
 
-    def enable_mocktime(self):
-        """Enable mocktime for the script.
-
-        mocktime may be needed for scripts that use the cached version of the
-        blockchain.  If the cached version of the blockchain is used without
-        mocktime then the mempools will not sync due to IBD.
-
-        For backward compatibility of the python scripts with previous
-        versions of the cache, this helper function sets mocktime to Jan 1,
-        2014 + (201 * 10 * 60)"""
-        self.mocktime = 1388534400 + (201 * 10 * 60)
-
-    def disable_mocktime(self):
-        self.mocktime = 0
-
     # Private helper methods. These should not be accessed by the subclass test scripts.
 
     def _start_logging(self):
@@ -451,6 +436,11 @@ class BitcoinTestFramework(metaclass=BitcoinTestMetaClass):
             for node in self.nodes:
                 node.wait_for_rpc_connection()
 
+            # For backward compatibility of the python scripts with previous
+            # versions of the cache, set mocktime to Jan 1,
+            # 2014 + (201 * 10 * 60)"""
+            self.mocktime = 1388534400 + (201 * 10 * 60)
+
             # Create a 200-block-long chain; each of the 4 first nodes
             # gets 25 mature blocks and 25 immature.
             # Note: To preserve compatibility with older versions of
@@ -458,7 +448,6 @@ class BitcoinTestFramework(metaclass=BitcoinTestMetaClass):
             #
             # blocks are created with timestamps 10 minutes apart
             # starting from 2010 minutes in the past
-            self.enable_mocktime()
             block_time = self.mocktime - (201 * 10 * 60)
             for i in range(2):
                 for peer in range(4):
@@ -472,7 +461,7 @@ class BitcoinTestFramework(metaclass=BitcoinTestMetaClass):
             # Shut them down, and clean up cache directories:
             self.stop_nodes()
             self.nodes = []
-            self.disable_mocktime()
+            self.mocktime = 0
 
             def cache_path(n, *paths):
                 return os.path.join(get_datadir_path(self.options.cachedir, n), "regtest", *paths)

--- a/test/functional/wallet_listtransactions.py
+++ b/test/functional/wallet_listtransactions.py
@@ -25,12 +25,13 @@ def tx_from_hex(hexstring):
 class ListTransactionsTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 2
-        self.enable_mocktime()
 
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
     def run_test(self):
+        self.nodes[0].generate(1)  # Get out of IBD
+        self.sync_all()
         # Simple send, 0 to 1:
         txid = self.nodes[0].sendtoaddress(self.nodes[1].getnewaddress(), 0.1)
         self.sync_all()

